scalableminds/webknossos
# Skeleton sanity check

### Mailable description of changes:
 - wK now checks if skeleton tracings are valid before you can upload them

### Steps to test:
- upload an invalid NML e.g. not connected tree, some nodes in edges don't exist
- get an error
- upload a valid NML
- upload should happen normally

### Issues:
- fixes #2108 

------
